README.md

```diff
@@ -1,22 +1,24 @@
+[![Travis](https://img.shields.io/travis/numpy/numpy.svg)](https://travis-ci.org/numpy/numpy) 
+
 NumPy is the fundamental package needed for scientific computing with Python.
 This package contains:
 
-    * a powerful N-dimensional array object
-    * sophisticated (broadcasting) functions
-    * tools for integrating C/C++ and Fortran code
-    * useful linear algebra, Fourier transform, and random number capabilities.
+   * a powerful N-dimensional array object
```

Comment:
Yes, lists are rendered indented regardless of the amount of space, unless it is 4 or more spaces, in which case it will be rendered as a code block.